tests/phpunit/tests/Stories.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Tests;
+
+use Google\Web_Stories\Story_Query as Testee;
+use Google\Web_Stories\Stories_Renderer\Generic_Renderer;
+use Google\Web_Stories\Story_Post_Type;
+
+/**
+ * @coversDefaultClass \Google\Web_Stories\Stories
```

Comment:
Since `Stories` was renamed to `Story_Query`, this should be changed as well.

Same goes for the file name.